stdlib/3/sys.pyi

```diff
@@ -122,7 +122,7 @@ def getcheckinterval() -> int: ...  # deprecated
 def getdefaultencoding() -> str: ...
 def getdlopenflags() -> int: ...  # Unix only
 def getfilesystemencoding() -> str: ...  # cannot return None
-def getrefcount(object) -> int: ...
+def getrefcount(object: Any) -> int: ...
```

Comment:
Done.

But changing the name of the argument will make a perfect valid python code fail (explicit kwargs)
```python
import sys

sys.getrefcount(object=1)
```
gives
```
asd2.py:3: error: Unexpected keyword argument "object" for "getrefcount"
```